### PR TITLE
renumber all our failure exit codes to be unique

### PR DESCRIPTION
**Pull request by cmr**
This might help us track down the mysterious `exit 1` we see. If it's still a `1`, we can dig deeper into our deps.

### CODE REVIEW

File: src/lib/transition_handler/processor.ml [hunk 1]
```diff
@@ -67,14 +62,12 @@ module Make (Inputs : Inputs.With_unprocessed_transition_cache.S) :
...
         ~catchup_job_writer ~catchup_breadcrumbs_writer
-        ~clean_up_signal:clean_up_catchup_scheduler
     in
     (* add a breadcrumb and perform post processing *)
     let add_and_finalize ~only_if_present cached_breadcrumb =
       let open Deferred.Or_error.Let_syntax in
       let%bind breadcrumb =
-        Deferred.Or_error.return
-          (Cached.invalidate_with_success cached_breadcrumb)
+        Deferred.return (Cached.invalidate cached_breadcrumb)
```
Comment (nholland94) [inline src/lib/transition_handler/processor.ml]: I don't see this function in the `Cached` interface. Shouldn't you use either `invalidate_with_success` or `invalidate_with_failure`?

**Comment by nholland94** [comment]
I'm confused by the changes to some interaction with `Cached`. It seems like you have regressed to the old `Cached` API somehow.

File: src/app/cli/src/client.ml [hunk 2]
```diff
@@ -20,7 +20,7 @@ let dispatch rpc query port =
    failing result. To deal with that, the success handler returns an
    Or_error. *)
 let dispatch_with_message rpc query port ~success ~error =
-  let fail err = eprintf "%s\n" err ; exit 1 in
+  let fail err = eprintf "%s\n%1" err ; exit 14 in
```
[the remark this answers is not in the document]
Comment (cmr) [reply src/app/cli/src/client.ml]: a typo of `%!`

File: src/lib/transition_handler/processor.ml [hunk 3]
```diff
@@ -29,9 +29,7 @@ module Make (Inputs : Inputs.With_unprocessed_transition_cache.S) :
   module Catchup_scheduler = Catchup_scheduler.Make (Inputs)
 
   (* TODO: calculate a sensible value from postake consensus arguments *)
-  let catchup_timeout_duration =
-    Time.Span.of_ms
-      (Consensus.Constants.block_window_duration_ms * 2 |> Int64.of_int)
+  let catchup_timeout_duration = Time.Span.of_ms 6000L
```
[the remark this answers is not in the document]
Comment (cmr) [reply src/lib/transition_handler/processor.ml]: These changes were stale, didn't realize the worktree wasn't clean when I started.

File: src/lib/cli_lib/background_daemon.ml [hunk 4]
```diff
@@ -23,7 +23,7 @@ let run ~f port arg =
     | Run_client ->
         f port arg
     | Abort ->
-        exit 1
```
[the remark this answers is not in the document]
Comment (cmr) [reply src/lib/cli_lib/background_daemon.ml]: fixed

**Comment by cmr** [comment]
it's still a 1.

**Pull request merged**